graylog2-web-interface/src/pages/ConfigurationsPage.test.tsx

```diff
@@ -33,28 +34,44 @@ jest.mock('stores/decorators/DecoratorsStore', () => ({
   DecoratorsStore: MockStore(),
 }));
 
-jest.mock('stores/configurations/ConfigurationsStore', () => MockStore(['getInitialState', () => ({})]));
+jest.mock('stores/configurations/ConfigurationsStore', () => MockStore(['getInitialState', () => ({
+  configuration: mockConfiguration,
+})]));
+
+jest.mock('actions/configurations/ConfigurationActions', () => ({
+  listWhiteListConfig: jest.fn(() => Promise.resolve()),
+  list: jest.fn(() => Promise.resolve()),
+  listMessageProcessorsConfig: jest.fn(() => Promise.resolve()),
+}));
+
+jest.mock('stores/streams/StreamsStore', () => ({
+  StreamsActions: { listStreams: () => Promise.resolve() },
+}));
 
 const ComponentThrowingError = () => {
-  throw Error('Boom!');
+  throw new Error('Boom!');
```

Comment:
Changing this is fine, but both are identical: `Thus the function call Error(…) is equivalent to the object creation expression new Error(…) with the same arguments.` ([Source](https://262.ecma-international.org/6.0/#sec-error-constructor))